Sources/Sentry/Public/SentryAppStartMeasurement.h

```diff
@@ -45,7 +45,7 @@ SENTRY_NO_INIT
 @property (readonly, nonatomic, assign) NSTimeInterval duration;
 
 /**
- * The timestamp when the app started, which is is the proces start timestamp.
+ * The timestamp when the app started, which is the process start timestamp.
```

Comment:
I made a separate PR for typos: https://github.com/getsentry/sentry-cocoa/pull/2174